### PR TITLE
Take into account the latest i18n/L10n correction report

### PR DESCRIPTION
**Pull request by niconoir**

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -3,8 +3,10 @@
     "addNewStudyDescription": "To add a new study, you should enter the following information:",
     "caseExist": "Does the case already exist?",
     "studyName": "study name",
+    "studyNameProperty": "study name: ",
     "studyDescription": "study description",
-    "owner": "Owner",
+    "studyDescriptionProperty": "study description: ",
```
Comment (SlimaneAmar) [inline src/translations/en.json:8]: case-sensitive is is taken into account

File: src/translations/en.json [hunk 2]
```diff
@@ -124,7 +126,7 @@
     "RatioTap3": "Ratio tap 3",
     "PhaseTap3": "Phase tap 3",
 
-    "viewOverloadsTable": "View overloads table",
+    "viewOverloadsTable": "View overload table: ",
```
Comment (SlimaneAmar) [inline src/translations/en.json:129]: There are several overloads

File: src/translations/en.json [hunk 3]
```diff
@@ -69,18 +71,18 @@
...
     "StaticArrows": "Static arrows",
     "AnimatedArrows": "Animated arrows",
     "Feeders": "Feeder",
-    "LineFlowColorMode": "Line color mode",
+    "LineFlowColorMode": "Line color mode: ",
     "NominalVoltage": "Nominal voltage",
     "Overloads": "Overloads",
     "Overload": "Overload",
-    "AlertThresholdLabel": "Alert threshold",
+    "AlertThresholdLabel": "Threshold for displaying lines\n before overload (%): ",
```
Comment (SlimaneAmar) [inline src/translations/en.json:85]: It seems that the newline is not taken into account ?

File: src/translations/en.json (hunk 2 above)
Comment (niconoir) [reply src/translations/en.json:129]: This change was made on purpose, it was required.

File: src/translations/en.json (hunk 3 above)
Comment (niconoir) [reply src/translations/en.json]: Yes, it was added for test purpose and I forgot to remove it. Removed.

File: src/translations/en.json (hunk 1 above)
Comment (niconoir) [reply src/translations/en.json]: I agree, added uppercases where necessary.

**Review by SlimaneAmar — APPROVED** [review]

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**